src/main/java/com/amazon/reviews/service/impl/CsvFileParsingServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package com.amazon.reviews.service.impl;
+
+import com.amazon.reviews.model.dto.ReviewDto;
+import com.amazon.reviews.service.FileParsingService;
+import com.amazon.reviews.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+
+public class CsvFileParsingServiceImpl implements FileParsingService {
+    private static final int ID_INDEX = 0;
+    private static final int PRODUCT_ID_INDEX = 1;
+    private static final int USER_ID_INDEX = 2;
+    private static final int PROFILE_NAME_INDEX = 3;
+    private static final int HELPFULNESS_NUMERATOR_INDEX = 4;
+    private static final int HELPFULNESS_DENOMINATOR_INDEX = 5;
+    private static final int SCORE_INDEX = 6;
+    private static final int TIME_INDEX = 7;
+    private static final int SUMMARY_INDEX = 8;
+    private final static int TEXT_INDEX = 9;
+    private final static int LAST_CSV_COMMA = 9;
+    private final static int PARSED_LINE_SIZE = 10;
+    private final FileReaderService fileReaderService;
+
+    public CsvFileParsingServiceImpl(FileReaderService fileReaderService) {
+        this.fileReaderService = fileReaderService;
+    }
+
+    @Override
+    public List<ReviewDto> parse(String path) {
```

Comment:
I think it is better to have a method that parses a single line and we can just use stream later to parse all the lines in the Service layer

Reply:
Also, this method should receive a string, not path

Reply:
> this method should receive a string, not path

I agree, this method should receive a single line of text